testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorSmokeTest.java

```diff
@@ -221,6 +222,24 @@ public void testRowLevelDelete()
         }
     }
 
+    @Test
+    public void testUpdate()
+    {
+        if (!hasBehavior(SUPPORTS_UPDATE)) {
+            // Note this change is a no-op, if actually run
+            assertQueryFails("UPDATE nation SET nationkey = nationkey + regionkey WHERE regionkey < 1", "This connector does not support updates");
```

Comment:
Can we in future omit such no-op operations in some connectors (hence silently breaking the test)?

Reply:
I am not a big fan on testing that on a logically immutable table. If update has bug and actually mutates the table we will get weird test failures in a very unrelated place

Reply:
> Can we in future omit such no-op operations in some connectors (hence silently breaking the test)?

@hashhar  That would require implementing some form of UPDATE support.

> If update has bug and actually mutates the table

@losipiuk in the worst case, the test env needs a fix. I don't think it's a big deal.